src/util.h

```diff
@@ -38,12 +38,6 @@ inline T* Malloc(size_t n);
 template <typename T>
 inline T* Calloc(size_t n);
 
-// Shortcuts for char*.
-inline char* Malloc(size_t n) { return Malloc<char>(n); }
-inline char* Calloc(size_t n) { return Calloc<char>(n); }
-inline char* UncheckedMalloc(size_t n) { return UncheckedMalloc<char>(n); }
-inline char* UncheckedCalloc(size_t n) { return UncheckedCalloc<char>(n); }
```

Comment:
I think what @bnoordhuis would like to see is declarations like `inline char* Malloc(size_t);` in the `util.h` header, as that’s where people look to see what the actual internal APIs are.